www/data_service/src/services/data/data.service.coffee

```diff
@@ -0,0 +1,146 @@
+# TODO rename this, or the dataService in waterfall view
+class Data extends Provider
+    constructor: ->
+    # TODO caching
+    cache: false
+
+    $get: ($log, $injector, $q, restService, socketService, helperService, ENDPOINTS) ->
+        return new class DataService
```

Comment:
will it see the injected services?

On Thu, Jun 11, 2015, 17:32 Pierre Tardy <notifications@github.com> wrote:

> In www/data_service/src/services/data/data.service.coffee
> <https://github.com/buildbot/buildbot/pull/1703#discussion_r32233480>:
>
> > @@ -0,0 +1,146 @@
> > +# TODO rename this, or the dataService in waterfall view
> > +class Data extends Provider
> > +    constructor: ->
> > +    # TODO caching
> > +    cache: false
> > +
> > +    $get: ($log, $injector, $q, restService, socketService, helperService, ENDPOINTS) ->
> > +        return new class DataService
>
> {{{
> class DataService
> self = null
> constructor: ->
> #[...]
>
> class Data extends Provider
> #...
>
>
> $get: ($log, $injector, $q, restService, socketService, helperService,
> ENDPOINTS) ->
>
> return new DataService()
> }}}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1703/files#r32233480>.
>